howto/enable-dapr-api-token-based-authentication/README.md

```diff
@@ -0,0 +1,93 @@
+# Enable Dapr APIs token-based authentication
+
+By default, Dapr relies on the network boundary to limit access to its public API. If you plan on exposing the Dapr API outside of that boundary, or if your deployment demands an additional level of security, consider enabling the token authentication for Dapr APIs. This will cause Dapr to require every incoming gRPC and HTTP request for its APIs for to include authentication token, before allowing that request to pass through. 
+
+## Creation of token
+
+Dapr uses [JWT](https://jwt.io/) tokens for API authentication. 
+
+> Note, while Dapr itself is actually not the JWT token issuer in this implementation, being explicit about the use of JWT standard enables federated implementations in the future (e.g. OAuth2).
+
+To configure APIs authentication, start by generating your token using any JWT token compatible tool (e.g. https://jwt.io/) and your secret. 
+
+> Note, that secret is only necessary to generate the token, and Dapr doesn't need to know about or store it
+
+## Configure API token authentication in Dapr
+
+The token authentication configuration is slightly different for either Kubernetes or self-hosted Dapr deployments: 
+ 
+### Self-hosted 
+
+In self-hosting scenario, Dapr looks for the presence of `DAPR_API_TOKEN` environment variable. If that environment variable is set while `daprd` process launches, Dapr will enforce authentication on its public APIs: 
+
+```shell
+export DAPR_API_TOKEN=<token>
+```
+
+To rotate the configured token, simply set the `DAPR_API_TOKEN` environment variable to the new value and restart the `daprd` process. 
```

Comment:
Maybe put this in a separate "Token rotation" section to address rotation for both deployment target? (this way we have sections for create / configure / rotate)

Reply:
created stand alone section for token rotation